src/site/components/merger-social-sco.html

```diff
@@ -0,0 +1,93 @@
+{% comment %}
+======================
+Social Media - R. Rail
```

Comment:
What's this?

Reply:
This file is a copy and past of https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/site/components/merger-social.html